src/ddmd/expression.d

```diff
@@ -4210,6 +4210,13 @@ extern (C++) final class DsymbolExp : Expression
             return e;
         }
 
+        if(ScopeDsymbol scs = s.isForwardingScopeDsymbol())
```

Comment:
Unsure about the purpose of this, especially since it is never executed by the test suite. Please document.